cross-chain/optimism/external/optimismGoerli/ArbitrumWormholeGateway.json

```diff
@@ -0,0 +1,3 @@
+{
+  "address": "0xe3e0511EEbD87F08FbaE4486419cb5dFB06e1343"
```

Comment:
Non-blocking but there are two things we should do:
1. Note in the README that this value is hardcoded and may need to be updated for testnet deployment.
2. Add mapping for the Optimism gateway in the Arbitrum module and add a similar note to the README.